### PR TITLE
Add replication config to server.properties

### PR DESCRIPTION
**Pull request by ravisingal**

### CODE REVIEW

File: helm/templates/configmap.yaml [hunk 1]
```diff
@@ -188,6 +188,65 @@ data:
...
+    {{- end }}
+
+    {{- if .Values.config.replicaFetchWaitMaxMs }}
+    # max wait time for each fetcher request issued by follower replicas
+    replica.fetch.wait.max.ms={{ int .Values.config.replicaFetchWaitMaxMs }}
+    {{- end }}
+
+    {{- if .Values.config.replicaHighWatermarkCheckpointIntervalMs }}
+    # The frequency with which the high watermark is saved out to disk
+    replica.high.watermark.checkpoint.interval.ms={{ int .Values.config.replicaHighWatermarkCheckpointIntervalMs }}
```
Comment (kotharironak) [inline helm/templates/configmap.yaml:221]: This should `long` as per Kafka doc. can you check?

File: helm/templates/configmap.yaml [hunk 2]
```diff
@@ -188,6 +188,65 @@ data:
...
+    {{- end }}
+
+    {{- if .Values.config.replicaLagTimeMaxMs }}
+    # If a follower hasn't sent any fetch requests or hasn't consumed up to the leaders log end offset for at least this time,
+    # the leader will remove the follower from isr
+    replica.lag.time.max.ms={{ int .Values.config.replicaLagTimeMaxMs }}
```
Comment (kotharironak) [inline helm/templates/configmap.yaml:227]: This should also be `long`, can you check this one too?

File: helm/templates/configmap.yaml (hunk 1 above)
Comment (ravisingal) [reply helm/templates/configmap.yaml:221]: `int` function in helm template convert string to number. It does not differentiate between int and long.

File: helm/templates/configmap.yaml (hunk 2 above)
Comment (ravisingal) [reply helm/templates/configmap.yaml:227]: see above,

File: helm/templates/configmap.yaml (hunk 1 above)
Comment (kotharironak) [reply helm/templates/configmap.yaml:221]: sounds good.

**Pull request merged**